wiki/en/Running-a-Server.md

```diff
@@ -19,87 +19,67 @@ permalink: "/wiki/Running-a-Server"
 
 
 
-## Do I need to run a server?
+## Do I need to run a Server?
 
-The short answer is **no**. For various reasons, you should try to use other people's servers, at least at first while getting your sound set up with the Jamulus client.
+The short answer is **no**. For various reasons, you should try to use Servers listed by the Directories, at least at first while getting your sound set up with the Jamulus client. You may also be able to use a Server hosted by a third party. Doing so will save you the trouble of setting one up yourself.  
```

Comment:
> Servers listed by the Directories

"Servers listed by the **built in** Directories".  At this point, we're assuming they're not doing anything special at all, so not just _any_ Directories, just the built in ones.

> You may also be able to use a Server hosted by a third party. Doing so will save you the trouble of setting one up yourself.

This doesn't really fit here - this is still discussing running a Client, i.e. _not_ run a Server at all.  (See the point that's below it.)